dev/tests/integration/testsuite/Magento/Customer/_files/customer_subscribe.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Need to create rollback for this fixture

See example:
https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/disable_guest_checkout_rollback.php

Also, please, add `TODO` with ticket number https://github.com/magento/graphql-ce/issues/167

Thanks